src/codecoverage/backend/requirements_frozen.txt

```diff
@@ -88,12 +94,15 @@ py==1.8.0
 pyasn1==0.4.5
 pyasn1-modules==0.2.4
 pycodestyle==2.5.0
+pycparser==2.19
 pyflakes==2.1.1
 Pygments==2.3.1
+pyOpenSSL==19.0.0
 pytest==4.3.1
 pytest-asyncio==0.10.0
 pytest-cov==2.6.1
 pytest-runner==4.4
+python37==interpreter
```

Comment:
?

Reply:
Oups. My sed regex to build the frozen requirements was a bit too eager